repositories/decisions_repository.go

```diff
@@ -434,15 +434,22 @@ func (repo *DecisionRepositoryImpl) rulesOfDecision(ctx context.Context, exec Ex
 		return nil, err
 	}
 
+	var columns []string
+	columns = append(columns, columnsNames("dr", dbmodels.SelectDecisionRuleColumn)...)
+	columns = append(columns, "r.formula_ast_expression")
+	query := NewQueryBuilder().
+		Select(columns...).
+		From(fmt.Sprintf("%s AS dr", dbmodels.TABLE_DECISION_RULE)).
+		LeftJoin(fmt.Sprintf("%s AS r ON r.id = dr.rule_id", dbmodels.TABLE_RULES)).
+		Where(squirrel.Eq{"dr.decision_id": decisionId}).
+		OrderBy("dr.id")
+
```

Comment:
TBH on this: inside the `DecisionById` impl, we do :
- get all rules exec (this is the code above)
- get the decision + join on case (if any)

So in the end, we will do 2 joins. Still this is two separated querries (one for rule exec, one for decision)